scripts/toaster_configuration

```diff
@@ -21,7 +21,7 @@ configured_layers () {
 }
 
 create_toaster_configuration() {
-DISTRO=$(grep "DISTRO =" "$BUILDDIR/conf/local.conf"  | awk {'print $3'} | sed -e s:\'::g)
+DISTRO=$(grep -m 1 "DISTRO =" "$BUILDDIR/conf/local.conf"  | awk {'print $3'} | sed -e s:\'::g)
```

Comment:
`grep -m` isn't mandated by posix. Please use a more portable construct. I'd recommend piping to `head -n 1`.